app/lib/contexts/SettingsContext.tsx

```diff
@@ -1,23 +1,54 @@
 import React, { createContext, useState } from 'react';
 import { LineType, LineTypes } from '../../components/viewer/ReplayLines';
 
+// eslint-disable-next-line no-shadow
```

Comment:
I'm a bit confused what it's complaining about here - I assume it doesn't like `CameraMode`?
What is that shadowing? I can't see any other `CameraMode`.

Seems like a false positive maybe? A comment explaining it would really help 😄